.circleci/config.yml

```diff
@@ -13,17 +13,17 @@ orb_promotion_filters: &orb_promotion_filters
     only: /^(major|minor|patch)-release-v\d+\.\d+\.\d+$/
 
 orbs:
-  aws-eks: circleci/aws-eks@dev:alpha
+  aws-eks: circleci/aws-eks@dev:hotfix
```

Comment:
Just commenting here to document our intent. This hotfix will be overwritten in the next PR #46